pkg/proto/common.go

```diff
@@ -159,6 +159,19 @@ func NetworkStrFromScheme(scheme Scheme) string {
 	return prefix + string(scheme)
 }
 
+func ParseSchemeFromStr(scheme string) (Scheme, error) {
```

Comment:
This function is quite limited. We should generate wallets for integration tests with `L` scheme and so on. Better to receive one letter as cli option.

Reply:
Totally agree.

Reply:
Even if receiving one letter, it must be `string` and we'll need to parse it into `proto.Scheme`. Otherwise, a user would have to enter `Uint` number like 87 ('W') and so on..

Reply:
Or we can implement flag.Value interface for `proto.Scheme` to make a custom flag for our specific case